### ISSUE TITLE
Editor not loading after fresh install

### ISSUE THREAD

**Issue by MatthiasKauer** [scragg0x/realms-wiki#185]
Hi,
I'm a bit confused about the installation of realms-wiki. After completing the installation, the editor is not working. Instead of the black editor (left) and the white preview (right) panel I can see in the demo, I only receive two white panels. Neither of these panels is writable.

I am testing in a new Ubuntu 16.04 box.
```
matthias@xubfiend:~/dvl/ansible-collection$ vagrant up
Bringing machine 'default' up with 'virtualbox' provider...
==> default: Importing base box 'geerlingguy/ubuntu1604'...
<....>
```
I have then taken the requirements from the README, setup a virtualenv and install realms-wiki.
```
vagrant@trusty:~$ sudo apt-get install -y python-pip python-dev libxml2-dev libxslt1-dev zlib1g-dev libffi-dev libyaml-dev libssl-dev libsasl2-dev libldap2-dev
vagrant@trusty:~$ sudo apt install virtualenv
vagrant@trusty:~$ export LC_ALL=C   # issues with DE locale in virtualenv (missing in minimal box?)
vagrant@trusty:~$ virtualenv --system-site-packages realms-venv
vagrant@trusty:~$ source realms-venv/bin/activate
(realms-venv) vagrant@trusty:~$ pip install realms-wiki
(realms-venv) vagrant@trusty:~$ realms-wiki start
```
Result:
![screenshot_2016-09-21_20-47-14](https://cloud.githubusercontent.com/assets/2436121/18724419/a304da06-803c-11e6-8888-120d099d22fc.png)
Interestingly, Chromium doesn't load the page at all.

Any idea what I could try differently?

Thanks in advance,
Matthias

**Comment by gazpachoking** [comment]
I think we had a packaging error for the last release that caused some static files not to be included. You can install from the github repo for now, or explicitly grab an older working version from pip until a fixed release comes out.

**Comment by MatthiasKauer** [comment]
Thanks for clearing this up :)
Which version would you suggest? The one I had was 0.9.1, I think.
```
C:\dvl>pip install realms-wiki==1.0   #some non-existing version to get a list.
Collecting realms-wiki==1.0
  Could not find a version that satisfies the requirement realms-wiki==1.0 (from versions: 0.3.22, 0.3.23, 0.3.24, 0.3.25, 0.3.26, 0.3.31, 0.4.0, 0.4.1, 0.4.3, 0.4.4, 0.4.5, 0.4.6, 0.4.7, 0.4.8, 0.5.0, 0.5.1, 0.5.2, 0.5.3, 0.5.4, 0.6.0, 0.6.1, 0.7.0, 0.8.0, 0.8.1, 0.9.0, 0.9.1)
No matching distribution found for realms-wiki==1.0
```
Is 0.9.0 a good idea or should I go back further? (Note that I'm only checking on Windows; installation will be on Linux.) Is there a changelog available that I haven't found yet?

**Comment by gazpachoking** [comment]
New release is made, does that work for you?

**Comment by MatthiasKauer** [comment]
Thank you for your feedback.

I did the above with
```
pip install realms-wiki==0.9.3
```
and now the editor seems to work like a charm.

I'm having some issues with the search, however. It only finds text from the page names.
Is that the desired behavior? Do I need to install and configure whoosh, or so, in addition, or is it another packaging issue?
I checked and, as far as I can tell, the demo at realms.io has the same behavior.
